### PR TITLE
Fix #1446: [Lake] Reintroduce subscriptions, and slots

### LINKED ISSUE #1446: [ETL] Reintroduce subscriptions and slots into ETL
**Is your feature request related to a problem? Please describe.**
New predictoor dashboard will require subscriptions... we'll likely be doing things with slots as well (and they already existed in GQL/lake previously)

**Describe the solution you'd like**
Re-introduce subscriptions and slots

**DoD:**
- [ ] raw subscriptions table + columns are available
- [ ] raw slots table + columns are available
- [ ] conftest/fixture has been updated to provide raw subscriptions sample data
- [ ] conftest/fixture has been updated to provide raw slots sample data
- [ ] sample data has been generated
- [ ] fixture tests and basic validations are in place

### PR DESCRIPTION
**Pull request by idiom-bytes**
## Description
Fixes #1446 

## Commit message
- [x] Updated GQLDF to include all tables (minus contracts)
- [x] Updated all logic to work w/ latest subgraph updates
- [x] Fixed tests
- [x] Updated sample data and fixtures for new test coverage
- [x] Updated** all ETL tests to work w/ latest fixture

### CODE REVIEW

File: pdr_backend/accuracy/app.py [hunk 1]
```diff
@@ -247,12 +248,33 @@ def calculate_timeframe_timestamps(
...
+    etl = ETL(ppss, gql_data_factory)
+    while True:
+        etl.do_etl()
+        threading.Event().wait(300)  # Wait for 5 minutes (300 seconds)
+
+
+@enforce_types
+def filter_func(seconds_per_epoch: int) -> Callable[[Any], bool]:
+    # filter out all the items that have seconds_per_epoch different that the specified parameter
+    return lambda item: int(item.get("seconds_per_epoch", 0)) == seconds_per_epoch
```
Comment (calina-c) [inline pdr_backend/accuracy/app.py:273]: `return int(item.get("seconds_per_epoch", 0)) == seconds_per_epoch` is enough.

lambda is a way to define anonymous functions but this function is named ("filter_func"). Using a lambda as an alias to a named function is moot.

File: pdr_backend/lake/table.py [hunk 2]
```diff
@@ -18,18 +18,20 @@
...
 
 @enforce_types
 def is_etl_table(table_name: str) -> bool:
     table_name = table_name.removeprefix("_")
     table_name = table_name.removeprefix("temp_")
-    table_name = table_name.removeprefix("_")
-    table_name = table_name.removeprefix("etl_")
-    table_name = table_name.removeprefix("temp_")
+
+    for _, member in TableType.__members__.items():
```
Comment (calina-c) [inline pdr_backend/lake/table.py:33]: I think you can iterate through values() instead of items()

File: pdr_backend/lake_info/overview.py [hunk 3]
```diff
@@ -284,7 +284,53 @@ def validate_lake_tables_no_duplicates(self) -> List[str]:
...
+        """
+
+        rows_df: pl.DataFrame = self.db.query_data(query_unmatched_payouts)
+
+        if rows_df is None or rows_df.shape[0] == 0:
+            logger.info("No unmatched payouts found in the lake.")
+            return violations
 
+        logger.info("Unmatched Payouts:\n%s", rows_df)
         return violations
```
Comment (calina-c) [inline pdr_backend/lake_info/overview.py:336]: shouldn't the violation(s) be added to the list of violations?

File: setup.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 # Installed by pip install pdr-backend
 # or pip install -e .
 install_requirements = [
-    "black==24.4.2",
+    "black==24.2.0",
```
Comment (calina-c) [inline setup.py:12]: please revert this change.

**Review by calina-c — CHANGES_REQUESTED** [review]

**Comment by idiom-bytes** [comment]
moved all logic over to https://github.com/oceanprotocol/pdr-backend/pull/1489 in order to simplify review & merge

**Pull request closed without merging**